browser/net/brave_common_static_redirect_network_delegate_helper_unittest.cc

```diff
@@ -75,4 +75,52 @@ TEST_F(BraveCommonStaticRedirectNetworkDelegateHelperTest,
   EXPECT_EQ(ret, net::OK);
 }
 
+TEST_F(BraveCommonStaticRedirectNetworkDelegateHelperTest,
+    RedirectChromecastDownload) {
+  net::TestDelegate test_delegate;
+  GURL url(
```

Comment:
how do we know this still works if the extension version changes?

Reply:
changed to `random_hash` and `random_version`, the pattern should catch all changes to `hash` and `version`